attendance/test/userdata_test.dart

```diff
@@ -41,20 +41,17 @@ void main() {
       expect(testUser.nativeName, 'someName');
     });
 
-    /*test('should reject names longer than 20 chars', () {
+    test('should reject names longer than 20 chars', () {
+      testUser.rename('');
       testUser.rename('This is a very long name');
-      bool isFit;
-      testUser.nativeName.length < 21 ? isFit = true : isFit = false;
 
-      expect(isFit, true);
-    });*/
+      expect(testUser.nativeName, isEmpty);
```

Comment:
@Bahaaib expect the thrown exception
assume you made it like `throw Exception('name is too long!');`
then 
```Dart
try {
    testUser.rename('This is a very long name');
    expect(1 ,2); // to fail the test
} catch(ex) {
    expect(ex.message, 'name is too long!');
}